dir.props

```diff
@@ -10,7 +10,7 @@
 
   <!-- Build Tools Versions -->
   <PropertyGroup>
-    <BuildToolsVersion>1.0.25-prerelease-00075</BuildToolsVersion>
+    <BuildToolsVersion>1.0.25-prerelease-00080</BuildToolsVersion>
```

Comment:
It looks like CoreFx has moved to version 83 since this PR was created.  I don't know why or the consequences of us being on 80 when they are on 83.